perllib/FixMyStreet/Cobrand/Hounslow.pm

```diff
@@ -0,0 +1,151 @@
+package FixMyStreet::Cobrand::Hounslow;
+use parent 'FixMyStreet::Cobrand::Whitelabel';
+
+use strict;
+use warnings;
+
+sub council_area_id { 2483 }
+sub council_area { 'Hounslow' }
+sub council_name { 'Hounslow Highways' }
+sub council_url { 'hounslow' }
+sub example_places { ( 'TW3 1SN', "Depot Road" ) }
+
+sub map_type { 'Hounslow' }
+
+sub base_url {
+    my $self = shift;
+    return $self->next::method() if FixMyStreet->config('STAGING_SITE');
+    return 'https://fms.hounslowhighways.org';
+}
+
+sub enter_postcode_text {
+    my ($self) = @_;
+    return "Enter a Hounslow street name and area, or postcode";
+}
+
+sub admin_user_domain { 'hounslowhighways.org' }
+
+sub disambiguate_location {
+    my $self = shift;
+    my $string = shift;
+
+    my $town = "Hounslow";
+
+    # Some specific Hounslow roads produce lots of geocoder results
+    # for the same road; this picks just one.
+    ( $string, $town ) = ( "TW3 4HR", "" ) if $string =~ /lampton\s+road/i;
+    ( $string, $town ) = ( "TW3 4AJ", "" ) if $string =~ /kingsley\s+road/i;
+    ( $string, $town ) = ( "TW3 1YQ", "" ) if $string =~ /stanborough\s+road/i;
+
+    return {
+        %{ $self->SUPER::disambiguate_location() },
+        string => $string,
+        centre => '51.468495,-0.366134',
+        town => $town,
+        bounds => [ 51.420739, -0.461502, 51.502850, -0.243443 ],
+    };
+}
+
+sub get_geocoder {
+    return 'OSM'; # default of Bing gives poor results, let's try overriding.
+}
+
+sub on_map_default_status { 'open' }
+
+sub contact_email {
+    my $self = shift;
+    return join( '@', 'enquiries', $self->council_url . 'highways.org' );
+}
+
+sub send_questionnaires { 0 }
+
+sub enable_category_groups { 1 }
+
+sub suggest_duplicates { 1 }
```

Comment:
This is a feature flag on master, so this line can be removed (and it added to config).